README.md

```diff
@@ -53,20 +53,22 @@ and must by specified with the `--os-client-config` flag.
 
 Other options as the binding address/port can by explored with the --help flag.
 
-By default the openstack\_exporter serves on port `0.0.0.0:9180` at the `/metrics` URL.
+By default the openstack\_exporter serves on port `0.0.0.0:9180` at the `/probe` URL.
```

Comment:
OK, please specify the difference here between /metrics and /probe and how to use both perhaps would be good to point out the prometheus official documentation as a reference for /probe.